prechecks.yml

```diff
@@ -32,6 +32,8 @@
 - hosts: tendrl_server
   tags:
     - production
+  user: root
```

Comment:
The `user` parameter was renamed to `remote_user`:
https://docs.ansible.com/ansible/2.3/playbooks_intro.html#hosts-and-users
Not sure, if we want to change it on all other places, but it would be probably the best approach.

Reply:
Good point, but I would rename this at once in a different commit.